src/iso19111/io.cpp

```diff
@@ -2964,6 +2964,7 @@ WKTParser::Private::buildConversion(const WKTNodeNNPtr &node,
         starts_with(methodName,  "PROJ-based operation method:")) {
         auto projString = methodName.substr(strlen("PROJ-based operation method: "));
         if (starts_with(projString, "+proj=")) {
+            projString += geodeticCRS->datum()->ellipsoid()->exportToPROJString(PROJStringFormatter::create().get());
```

Comment:
adding a space char between both would probably be needed. And for exotic geodetic CRS datum() might be NULL. You want to directly use geodeticCRS->ellipsoid() that will fetch the ellipsoid for the datumEnsemble() when datum() is NULL

Reply:
That is really good to know - thanks for the tips!